website/content/docs/v0.1/Configuration/serverclasses.md

```diff
@@ -5,29 +5,60 @@ weight: 3
 
 # Server Classes
 
-Server classes are a way to group distinct server resources.
-The "qualifiers" key allows the administrator to specify criteria upon which to group these servers.
-There are currently three keys: `cpu`, `systemInformation`, and `labelSelectors`.
-Each of these keys accepts a list of entries.
-The top level keys are a "logical AND", while the lists under each key are a "logical OR".
-Qualifiers that are not specified are not evaluated.
+Server classes are a way to group distinct server resources.  The `qualifiers`
+and `selector` keys allow the administrator to specify criteria upon which to
+group these servers. If both of these keys are missing, the server class
+matches all servers that it is watching.  If both of these keys define
+requirements, these requirements are combined (logical `AND`).
+
+## `selector`
+
+`selector` groups server resources by their labels. The [Kubernetes
+documentation][label-selector-docs] has more information on how to use this
+field.
+
+## `qualifiers`
+
+There are currently four keys: `cpu`, `systemInformation`, `labelSelectors`,
+and `labelSelector`.  Each of these keys accepts a list of entries.  The top
```

Comment:
`labelSelector` key is not there, and I think we should stop documenting `labelSelectors` to make it less confusing. 

whoever is using `labelSelectors`, they will keep using it, but we don't want new users of this field.